arcgis-ios-sdk-samples/Augmented reality/Explore scenes in flyover AR/README.md

```diff
@@ -38,7 +38,7 @@ This sample requires a device that is compatible with ARKit 1.0 on iOS.
 
 **Flyover AR** is one of three main patterns for working with geographic information in augmented reality. See the [guide doc]() for more information.
```

Comment:
Does the `guide doc` refers to itself? Should I just leave it blank? As I saw Java/Android do the same thing.

Reply:
Definitely an oversight on Java/Android's part. Good catch. It was left blank so that each SDK could fill in the link with the appropriate value. For iOS, it should read:

```suggestion
**Flyover AR** is one of three main patterns for working with geographic information in augmented reality. See the [guide doc](https://developers.arcgis.com/ios/latest/swift/guide/display-scenes-in-augmented-reality.htm) for more information.
```